src/Chapter2.hs

```diff
@@ -39,6 +39,8 @@ Now, if you are ready, bring it on!
 -}
 
 module Chapter2 where
+import Data.List
+import Data.Map (Map)
```

Comment:
I wonder if this import is needed :thinking: 
Also, I'm not sure that it compiles... The `Data.Map` module is in the `containers` library, but we depend only on `base` (the standard library). Unfortunately, container types are not in the standard Haskell library :disappointed: